smartcontractkit/smart-contract-db
# feat: adds sub sections for communities

# Overview
Adds sub-sections for communities

# Screenshots for Mobile and Desktop views (if applicable)
Homepage:
![image](https://user-images.githubusercontent.com/20880360/138748990-d1839138-bb06-4425-98c1-92059c1a537f.png)

Communities page:
![image](https://user-images.githubusercontent.com/20880360/138749079-8c549fdd-bf57-43ae-bc0b-96043a882f31.png)

### Manual Testing

Write the steps required for how might test this. Example:

1. Run `npm run dev`
2. In browser, goto `localhost:3000/`
3. Scroll down on the homepage and verify the community sub-sections
4. Navigate to the Community page and verify the community sub-sections
5. Get results :tada:
